src/common/modify-settings-modal/index.tsx

```diff
@@ -358,24 +365,42 @@ const ModifySettingsModal = ({ position, open, onCancel }: ModifySettingsModalPr
     variant?: 'text' | 'outlined' | 'contained';
   }[] = [];
 
+  let Component;
+
   if (needsToApprove) {
-    actions = [
-      {
-        color: 'primary',
-        variant: 'contained',
-        label: (
+    Component = (
+      <SplitButton
```

Comment:
Can we also add a help tooltip icon to the main button like we had on the button before it?

The copy would be 
```
You must give the Mean Finance smart contracts permission to use your {token}
```